src/style.css

```
@@ -1 +1,65 @@
 /* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 14px;
+}
+
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav__list {
+  display: flex;
+  padding-left: 0;
+  margin: 0;
+}
+
+.nav__item {
+  list-style-type: none;
+  margin-right: 20px;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+  display: inline-block;
```

Why inline-block? You already have a flex wrapper, that's unnecessary 
```suggestion
  display: block;
```